SPECS/kernel/config

```diff
@@ -1157,7 +1157,6 @@ CONFIG_SKB_EXTENSIONS=y
 CONFIG_PACKET=y
 CONFIG_PACKET_DIAG=m
 CONFIG_UNIX=y
-CONFIG_UNIX_SCM=y
```

Comment:
Result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/8eb55b667a0e2dba4b2409cfffc87080724d54a6